### PR TITLE
Dependency upgrades + require sbt 1.9.7 as minimum

### PR DESCRIPTION
**Pull request by mkurz**

### CODE REVIEW

**Comment by mkurz** [comment]
Without increasing sbt to 1.9.7 we get errors like following (it seems some of the upgraded libraries are incompatible with older sbt):

```
[info] Running paradox/can-use-paradox
...
[info] [info] loading settings for project sbt_8fd99a15 from build.sbt ...
Error:  java.lang.NoSuchMethodError: 'java.io.File sbt.Defaults$.makeCrossTarget(java.io.File, java.lang.String, java.lang.String, java.lang.String, boolean, boolean)'
Error:  	at com.typesafe.sbt.web.SbtWeb$.$anonfun$buildSettings$6(SbtWeb.scala:153)
Error:  	at scala.Function1.$anonfun$compose$1(Function1.scala:49)
Error:  	at sbt.internal.util.EvaluateSettings$MixedNode.evaluate0(INode.scala:228)
Error:  	at sbt.internal.util.EvaluateSettings$INode.evaluate(INode.scala:170)
Error:  	at sbt.internal.util.EvaluateSettings.$anonfun$submitEvaluate$1(INode.scala:87)
Error:  	at sbt.internal.util.EvaluateSettings.sbt$internal$util$EvaluateSettings$$run0(INode.scala:99)
Error:  	at sbt.internal.util.EvaluateSettings$$anon$3.run(INode.scala:94)
Error:  	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
Error:  	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at sbt.scriptedtest.ScriptedTests.runOrHandleDisabled(ScriptedTests.scala:266)
Error:  	at java.base/java.lang.Thread.run(Thread.java:829)
[info] [error] java.lang.NoSuchMethodError: 'java.io.File sbt.Defaults$.makeCrossTarget(java.io.File, java.lang.String, java.lang.String, java.lang.String, boolean, boolean)'
[info] [error] Use 'last' for the full log.
...
Error:  x paradox/can-use-paradox 
	at sbt.scriptedtest.ScriptedTests.$anonfun$runBatchedTests$1(ScriptedTests.scala:236)
	at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:286)
	at scala.collection.mutable.ArraySeq.foreach(ArraySeq.scala:75)
	at scala.collection.TraversableLike.map(TraversableLike.scala:286)
	at scala.collection.TraversableLike.map$(TraversableLike.scala:279)
	at scala.collection.AbstractTraversable.map(Traversable.scala:108)
	at sbt.scriptedtest.ScriptedTests.runBatchTests$1(ScriptedTests.scala:208)
...
[error] (core / scripted) Failed tests:
[error] 	scaladoc/can-aggregate
[error] 	site/can-have-custom-mappings
[error] 	site/can-run-generator-twice
[error] 	site/minimum-sbt-1.0
[error] (paradox / scripted) Failed tests:
[error] 	paradox/can-use-paradox
```

**Comment by mkurz** [comment]
Ready.

However it seems the required workflows are misconfigured (because below workflow, with this pr, does not exist anymore):
image

**Review by ennru — APPROVED** [review]
LGTM.

File: build.sbt [hunk 1]
```diff
@@ -67,11 +67,6 @@ lazy val core = project
       "ws.unfiltered" %% "unfiltered-filter" % unfilteredVersion,
       "ws.unfiltered" %% "unfiltered-jetty" % unfilteredVersion,
       "ws.unfiltered" %% "unfiltered-specs2" % unfilteredVersion % Test,
-    ).map(
```
Comment (mdedetrich) [inline build.sbt:70]: @mkurz This is causing a regression, see https://github.com/apache/pekko-persistence-cassandra/actions/runs/8405372473/job/23018005449?pr=176

Comment (mkurz) [reply build.sbt:70]: Sorry! Seems this is now fixed via
- #217

**Pull request merged**